ic-os/guestos/context/docker-base.prod

```diff
@@ -1 +1 @@
-ghcr.io/dfinity/guestos-base@sha256:0a485d2d033df5aa7efd4ccbaad432c29d18f0f165cdcbeed5d49790c0d16340
+ghcr.io/dfinity/guestos-base@sha256:b3d3a65a8b5d2d45e2b085e52bf995e53e5bb592984c8229b98d6eb74dec22e0
```

Comment:
This contains changes for the GuestOS 24.04 upgrade, we'll keep this in an RC for the initial rollout